src/rgw/rgw_reshard.cc

```diff
@@ -651,6 +654,8 @@ int RGWBucketReshard::do_reshard(int num_shards,
 	  return ret;
 	}
 
+  if (ret = f.check("after_target_shard_entry"); ret < 0) { return ret; }
+
```

Comment:
these before/after checks don't appear to be used in testing, are they necessary? p.s. the indendation looks off

Reply:
no looks like you're correct, i will remove them